### PR TITLE
Bioinformatics Course > First Capstone Project

### PR DESCRIPTION
**Pull request by shahedakhatoon**
## Added the following:
1. Home page
2. About page
3. Presenters section dynamically
4. Configured Linters
5. Designed the UI with CSS.
6. Deployed the branch using netlify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+## Run
+
+> To run the above project :
+> Run the following commands on your terminal of choice
+
+- git clone <url>
+- cd /dir
+- open the project in vscode.
+
```
Comment (micheaol) [inline README.md:45]: Please ensure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      <div class="card">
+        <img class="program-img" src="src/exhibition.png" alt="Exhibition">
+        <div class="program-title"><h3>Exhibition</h3></div>
+        <p class="program-des">Appriciate various innovations, tehnologies and new ideas from various AI fields</p>
+      </div>
+      <div class="card">
+        <img class="program-img" src="src/forum.png" alt="Forum">
+        <div class="program-title"><h3>Forum</h3></div>
+        <p class="program-des">Have the time to share our thoughts and opinions with experts for each topic</p>
+      </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the margin between the cards looks almost identical to the original design according to the requirement.

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    <button class="join-btn" type="button">Join Online Course</button>
+    <a class="wholeProgram" href="#">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section class="lecturers">
+    <h2 class="lecturerTitle">Lecturers</h2>
+      <div class="red-line"></div>
+    <div class="lecturersCards-preview"></div>
+    <div class="lecturersCards">
+    </div>
+    <button class="more-btn" type="button">More <span> <i class="fa fa-angle-down"></i></span></button>
```
Comment (micheaol) [inline index.html:92]: Please ensure that this section uses the exact same background color, borders, etc, which the cards border invisible from the original design.

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        <p class="program-des">Have the time to share our thoughts and opinions with experts for each topic</p>
+      </div>
+      <div class="card">
+        <img class="program-img" src="src/workshop.png" alt="workshop">
+        <div class="program-title"><h3>Workshop</h3></div>
+        <p class="program-des">Try creating your own work using open source licenses rather than just whatching</p>
+      </div>
+      <div class="card">
+        <img class="program-img" src="src/ignite.png" alt="Networking">
+        <div class="program-title"><h3>Networking</h3></div>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
Home_mobile | image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+        <div class="program-title"><h3>Lecture</h3></div>
+        <p class="program-des">Listen to all speakers from various regions about the machin learning algorithms</p>
+      </div>
+      <div class="card">
+        <img class="program-img" src="src/exhibition.png" alt="Exhibition">
+        <div class="program-title"><h3>Exhibition</h3></div>
+        <p class="program-des">Appriciate various innovations, tehnologies and new ideas from various AI fields</p>
+      </div>
+      <div class="card">
+        <img class="program-img" src="src/forum.png" alt="Forum">
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
image | image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    </div>
+  </header>
+<section class="about">
+  <h1 class="aboutTitle">BIOINFORMATICS<br>ONLINE COURSES 2022</h1>
+  <p class="abou-textBox">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed molestie ante, a semper magna. Nam quis iaculis libero. Sed vehicula, augue et dapibus ornare, magna risus aliquet ipsum, in imperdiet ex arcu ac risus. Cras porta aliquam porttitor. Nunc sed tortor ipsum. Cras suscipit ex risus, sed consequat quam aliquet id. Cras hendrerit suscipit odio, sit amet venenatis eros vestibulum consectetur. Integer at enim sagittis, lacinia massa nec, condimentum ex.</p>
+  <p class="about-normal-text">Please contact us per Email for any further question about Computational Biology online coures !</p>
+  <a id="about-link" href="#">bioinformatics2022@gmail.com <img src="https://img.icons8.com/external-those-icons-lineal-those-icons/10/000000/external-right-arrows-those-icons-lineal-those-icons-1.png" alt="close"/></a>
+</section>
+<section class="logo-section">
+  <h2 class="lecturerTitle">Bioinformatics Online Course 2022</h2>
```
Comment (micheaol) [inline about.html:53]: Please ensure that your page is responsive according to the requirement.

image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  <div class="logo-border"><img id="ml-logo" src="src/bioinformatics-largelogo.png" alt="courseLogo"></div>
+</section>
+<section class="gallery">
+  <h2 class="lecturerTitle">See The past courses </h2>
+  <div class="red-line"></div>
+  <p class="about-normal-text" id="gallery-caption">Take a look at the last two Bioinformatics workshop</p>
+  <div class="photoes">
+    <div class="photo-fram">
+    <img src="src/workshop1.jpg" class="photo" alt="workshop2021">
+      <div class="layer">
```
Comment (micheaol) [inline about.html:65]: Please ensure that the pages should look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
image | image

File: README.md [hunk 8]
```diff
@@ -1 +1,46 @@
...
+- Html
+- Css
+- Js
+- Git & Gitflow
+- [netlify](https://www.netlify.com/) for deployment
+
+## [Visit the website](https://bioinformatics-course.netlify.app/)
+
+## Live Demo
```
Comment (micheaol) [inline README.md]: Please ensure that you make use of GitHub Pages to deploy the web pages according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @shahedakhatoon , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (shahedakhatoon) [reply index.html:92]: @micheaol Please be clear about the word invisible, applying the color swatch #d3d3d3 given in the design doesn't look similar to the lecturers' card in the Figma. In fact, the background color and border that I've used look more similar to the Figma design. Please see the screenshot, after changing background to #d3d3d3, it looks dissimilar.
127 0 0 1_5500_Bioinformatics-online-courses_

File: index.html (hunk 4 above)
Comment (shahedakhatoon) [reply index.html]: @micheaol Please see the changes

127 0 0 1_5500_Bioinformatics-online-courses_ (1)

File: about.html (hunk 6 above)
Comment (shahedakhatoon) [reply about.html:53]: @micheaol Please see the changes I've made. I find he title responsive now

127 0 0 1_5500_Bioinformatics-online-courses_about html

File: index.html (hunk 2 above)
Comment (shahedakhatoon) [reply index.html]: @micheaol Okay! I've removed the gap here, I think it looks exactly the same now. Please see the screenshot.

127 0 0 1_5500_Bioinformatics-online-courses_index html (2)

File: index.html (hunk 3 above)
Comment (micheaol) [reply index.html:92]: @shahedakhatoon 
Thank you for making other changes. To summarize, please make the card the same color as the background to match the original design.

File: index.html (hunk 5 above)
Comment (shahedakhatoon) [reply index.html]: @micheaol I've made the changes you requested, and I also think the Figma template is that of a Mobile-L view. I have used flexbox and grid properties to place the items in the partners cards. Please see the screenshot

127 0 0 1_5500_Bioinformatics-online-courses_index html (3)

File: about.html (hunk 7 above)
Comment (shahedakhatoon) [reply about.html:65]: @micheaol I've made the changes you requested.

127 0 0 1_5500_Bioinformatics-online-courses_about html (1)

File: index.html (hunk 3 above)
Comment (shahedakhatoon) [reply index.html:92]: @micheaol Okay, thank you. I've set the card the same color as the background,i.e. rgb(247,247,247)

127 0 0 1_5500_Bioinformatics-online-courses_index html (4)

File: README.md (hunk 8 above)
Comment (shahedakhatoon) [reply README.md]: @micheaol Okay I've deployed on Github pages but the changes I've made did not get deployed

Comment (micheaol) [reply README.md]: @shahedakhatoon 
Please give it some time and check again for the changes.

Comment (shahedakhatoon) [reply README.md]: okay

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @shahedakhatoon , 

Great work on making the changes requested by the previous reviewer 👏 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

well-1
Good luck! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**